boards/arm/mimxrt1062_fmurt6/mimxrt1062_fmurt6.dts

```diff
@@ -20,12 +20,39 @@
 		led1 = &red_led;
 		watchdog0 = &wdog0;
 		sdhc0 = &usdhc1;
-		pwm-0 = &flexpwm2_pwm0;
 		gps1 = &lpuart2;
 		telem1 = &lpuart3;
 		telem4-gps2 = &lpuart5;
 	};
 
+	pwm_shell: pwm_shell {
```

Comment:
the commit title `MIMXRT1062-FMURT6: Fix PWM device names.` Should be updated to something like the following:
`soc: arm: nxp_imx: fix PWM node names`